### DISCUSSION TITLE
sdcard and LCD share a set of SPI, how to deal with the conflict between them?

### DISCUSSION THREAD

**Discussion by lbuque** [micropython/micropython#11773] [ESP32]
I am currently using M5CoreS3, its SDCard and LCD share a set of SPI, is there any good way to use them at the same time?

![image](https://github.com/micropython/micropython/assets/46518962/b46c94ad-db92-432a-9e38-83e70825ff4d)

**Comment by davefes** [comment]
They use separate CS GPIOs so you can talk to the SDcard or the LCD.  If you want both to appear to be working at the same time you will need to investigate `uasyncio`
